.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -1,34 +1,54 @@
 ---
 name: Bug report
 about: Create a report to help us improve
-
 ---
 
-**Please make sure to read the [FAQ](https://github.com/Alf21/TTT2/wiki/faq) first!**
+## Disclaimer
+
+If you are missing the following information your Issue will be closed!
+
+- "Your Version of TTT2"
+- "Describe the Bug"
+
+Also please make sure to read the [FAQ](https://github.com/TTT-2/TTT2/wiki/faq) first!
+
+## Your Version of TTT2 (mandatory)
+
+Do you use the Workshop Version of TTT2 or did you pull a Version from GitHub?  
+If the latter is the case please also provide the (shortened) commithash.  
+(executing `git log -n 1 --pretty=format:"%h"` in your Servers TTT2 repository will provide it to you)
```

Comment:
This is not always the case, as most people do not clone the repository and rather download the zip file. I think adding a hint, that this command is only possible when the repository was cloned would be enough. Otherwise a commit hash can only be given, if the user still knows when and what commit he downloaded.

Reply:
I actually have not thought about people getting the zip ... 
We definitely need some clever and intuitive way to provide the versioning.

I also thought about asking to update to the latest version first before submitting an issue but then i would be worried that some bugs will just not be reported as fast (or not at all) and hence will be missed.